modules/migration-compatibility-guidelines.adoc

```diff
@@ -20,22 +20,25 @@ modern operator:: The {mtc-short} Operator designed for modern platforms.
 control cluster:: The cluster that runs the {mtc-short} controller and GUI.
 remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
 
-
-[cols="1,2,2", options="header"]
 .{mtc-short} compatibility: Migrating from a legacy platform
 |===
-||{product-title} 4.5 or earlier |{product-title} 4.6 or later
-|Stable {mtc-short} version a|{mtc-short} {mtc-version}._z_
+| |{product-title} 3.11 |{product-title} 4.0 to 4.5 |{product-title} 4.6 or later
+
+|Stable {mtc-short} version
+|{mtc-short} {mtc-version}v.1.7
+|{mtc-short} {mtc-version}._z_
```

Comment:
Here also 1.8 is mentioned, while 1.8 is only for 4.10+ clusters.

Reply:
@stillalearner 
This is what was in the documentation beforehand. I only added the 3.11 column. 
@dymurray 
Should this whole table be updated?

Reply:
Ah sorry I misunderstood the table first review... 

if you are migrating from OCP 3.11 to OCP 4, you need MTC 1.7 on both the source and target clusters. You cannot use MTC 1.8 in this scenario at all.

1.8 Is only usable when source and target clusters are OCP 4